renderer/index.js

```diff
@@ -409,7 +415,28 @@ function jumpToTime (time) {
     state.playing.jumpToTime = time
   }
 }
-
+function setPlaybackRate (direction) {
+  var rate = state.playing.playbackRate
+  if (direction > 0 && rate >= 0.25 && rate < 2) {
+    rate += 0.25
+  } else if (direction < 0 && rate > 0.25 && rate <= 2) {
+    rate -= 0.25
+  } else if (direction < 0 && rate === 0.25) { /* when we set playback rate at 0 in html 5, playback hangs ;( */
+    rate = -1
+  } else if (direction > 0 && rate === -1) {
+    rate = 0.25
+  } else if ((direction > 0 && rate >= 1 && rate < 16) || (direction < 0 && rate > -16 && rate <= -1)) {
+    rate *= 2
+  } else if ((direction < 0 && rate > 1 && rate <= 16) || (direction > 0 && rate >= -16 && rate < -1)) {
+    rate /= 2
+  } else if (direction < 0 && rate === 1) {
+    rate = -1
+  }
+  state.playing.playbackRate = rate
+  if (lazyLoadCast().isCasting()) {
+    Cast.setRate(rate)
```

Comment:
See below --- when `Cast.setRate` returns false, let's set `rate = 1`.

That way, the increase/decrease playback rate won't do anything on Chromecast (instead of showing a label that says "2X" or "0.5X" while the actual playback speed doesn't change)